saltstack/base/salt/cis-controls/init.sls

```diff
@@ -0,0 +1,22 @@
+#CentOS 1.1.1 Disable unused filesystems
+#https://jira.cloudera.com/browse/CB-8897
```

Comment:
Probably, it's worth to safeguard these changes to make them only on redhat7 based systems. We have lots of similar things in the code for debian,ubuntu,suse. Right now we don't support these, but it's a small investment that might worth it later.

Reply:
Added target as centos7 for these changes.